index.js

```diff
@@ -105,13 +106,15 @@ const run = async () => {
         // Create Issue
         createdIssues.push({ title, body, labels })
       } else {
-        title = `${issueTitlePrefix}${new Date().toTimeString()}`
-        createdIssues[0].title = title
-
+        counter++
```

Comment:
Maybe we should move this out of the else. Otherwise we do't count the first item.

Reply:
Aargh... You're right!

Reply:
Done.